config-example.py

```diff
@@ -22,3 +22,9 @@
 # Name of the S3 bucket used to import employee data from a file named employees.json
 # Check out /import/employees.json.example to see how this file should look like.
 S3_BUCKET = 'employees'
+
+# When do we use Gravatar? Options are:
+# * 'always' - prefers Gravatar over the Employee.photo_url
+# * 'backup' - use Gravatar when photo_url is empty
+# * anything else - disabled
+GRAVATAR = 'always'
```

Comment:
I'd vote for 'backup' as the example / default, as I'd expect most people would want to have the company-specific photo override gravatar, if one is available.